api/api_private_account.go

```diff
@@ -485,31 +485,29 @@ func (s *PrivateAccountAPI) Sign(ctx context.Context, data hexutil.Bytes, addr c
 //
 // https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal_ecRecover
 func (s *PrivateAccountAPI) EcRecover(ctx context.Context, data, sig hexutil.Bytes) (common.Address, error) {
-	pubkey, err := klayEthEcRecover(data, sig)
+	if len(sig) != crypto.SignatureLength {
+		return common.Address{}, fmt.Errorf("signature must be 65 bytes long")
+	}
+	if sig[crypto.RecoveryIDOffset] != 27 && sig[crypto.RecoveryIDOffset] != 28 {
+		return common.Address{}, fmt.Errorf("invalid Klaytn signature (V is not 27 or 28)")
+	}
+
+	// Transform yellow paper V from 27/28 to 0/1
+	sig[crypto.RecoveryIDOffset] -= 27
+
+	pubkey, err := klayEcRecover(data, sig)
```

Comment:
Why this API was reverted back not to support `ethEcRecover`?

Reply:
We cannot identify if the signature has signed with klay or eth prefix without the signer's address.
Even though a user signed message with eth prefix, it will return something. that's why we can verify only klay prefixed signature with this function.

Reply:
Thanks. I am gonna leave this comment open for other reviewer.